.github/workflows/register-buildpack.yml

```diff
@@ -36,7 +36,15 @@ jobs:
               process.exit(1)
             }
             console.log(`::set-env name=BUILDPACK::${JSON.stringify(result)}`)
-
+            console.log(`::set-output name=namespace::${result.ns}`)
+            console.log(`::set-output name=name::${result.name}`)
+            console.log(`::set-output name=version::${result.version}`)
+            console.log(`::set-output name=address::${result.addr}`)
+      - uses: docker://ghcr.io/buildpacks/actions/verify-buildpackage
```

Comment:
```suggestion
      - uses: docker://ghcr.io/buildpacks/actions/verify-buildpackage:main
```

Reply:
I might need to update all the docs in `buildpacks/github-actions` with this.  It's not intuitive that you need a version.